src/display/editor/stamp.js

```diff
@@ -423,6 +427,48 @@ class StampEditor extends AnnotationEditor {
     return bitmap;
   }
 
+  async #mlGuessAltText(bitmap, width, height) {
+    if (this.parent && this.#guessedAltText !== null) {
+      if (this.#guessedAltText) {
+        this.altTextData = { altText: this.#guessedAltText, decorative: false };
+      }
+      return;
+    }
```

Comment:
Isn't this a bit redundant, since if `this.#guessedAltText === ""` then we'd return early anyway just below since `this.#hasMLBeenQueried === true` in that case?

However, should we perhaps also check that no altText already exists here to avoid any risk of overwriting something?

```suggestion
    if (this.parent && this.#guessedAltText && !this.hasAltText()) {
      this.altTextData = { altText: this.#guessedAltText, decorative: false };
      return;
    }
```